corehq/messaging/smsbackends/vertex/forms.py

```diff
@@ -0,0 +1,36 @@
+from corehq.apps.sms.forms import BackendForm
+from django import forms
+from crispy_forms import layout as crispy
+from django.utils.translation import ugettext_lazy, ugettext as _
+
+
+class VertexBackendForm(BackendForm):
+    _cchq_domain = "enikshay"
```

Comment:
this isn't necessary - we'll just create an instance of this gateway in the domain we need so it's only available there